### ISSUE TITLE
Code examples are not readable on a kindle

### ISSUE THREAD

**Issue by jaseemabid** [addyosmani/backbone-fundamentals#532]
The code comments and strings have  a very light grey colour and its not readable at all. 

![2014-01-04 13 35 52](https://f.cloud.github.com/assets/601714/1844340/c9105140-751c-11e3-9fe6-f2b7b83e204b.jpg)

This is a code example compared with standard text for reference. 

![2014-01-04 13 37 19](https://f.cloud.github.com/assets/601714/1844339/c778526a-751c-11e3-86d3-5e61fb43438d.jpg)

Excuse bad photography :/

**Comment by sunorry** [comment]
@jaseemabid I use kpw2, and it can't read the file. I don't know why.

**Comment by jaseemabid** [comment]
I think we should just use the same black colour, but just _italicise_ comments and **bold** keywords and strings.

**Comment by addyosmani** [comment]
Agreed. Will keep this open and try to see if we can get a better CSS override for the Kindle build in place.
